src/index.html

```diff
@@ -1,13 +1,53 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        class="card__image"
+        src="images/imac.jpeg"
+        alt="product image"
```

Comment:
Perhaps it is better to give a more precise name
```suggestion
        alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
```